src/Curl.Parser.Net/CurlOptions.cs

```diff
@@ -60,7 +62,7 @@ public CurlOptions()
 
     public string KeyFileName { get; set; }
 
-    public KeyType KeyType { get; set; }
+    public CertificateType KeyType { get; set; }
```

Comment:
This is actually no the same.
KeyType cannot be P12. So I suggest to keep both enums.

Reply:
I see. Done updating.